### DISCUSSION TITLE
Wdio Helper Wizard - set configuration without manual intervention

### DISCUSSION THREAD

**Discussion by ellatom** [webdriverio/webdriverio#8284] [Technical Discussion]
Hi, 

I would like to run webdriverio projects with the same configurations. Each time when I create a new wdio project, I need an options to select on wdio CLI options not manually but via script or configuration.
This way an automation can be installed on VM and  run without manual intervention.

Is there an option for that ?
I found that someone already asked for this option :https://github.com/webdriverio/webdriverio/issues/6765
Thanks

**Comment by christian-bromann** [comment]
> I need an options to select on wdio CLI options not manually but via script or configuration.

This is not possible atm. Can you explain how exactly this solution should look like and a bit about your use case?
